app/_components/protocol/template.njk

```diff
@@ -0,0 +1,141 @@
+{%- from "../summary-card/macro.njk" import appSummaryCard with context -%}
+{%- from "../document-list/macro.njk" import appDocumentList with context -%}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+### {{ params.title }}
+
+{%- set apiHtml -%}
+  {%- if params.items.api.href -%}
+    <a href="{{- params.items.api.href -}}"><code>{{ params.items.api.param | nl2br | safe }}</code></a>
```

Comment:
The grey background on this hides the focus state somewhat.

Reply:
Could make the link to docs more explicit, and separate from the code block?

Reply:
Made link explicit.